client/lib/abtest/active-tests.js

```diff
@@ -9,6 +9,15 @@
 /* See /client/components/experiment/readme.md for more info!
 /**************************************************************************************************/
 
+/**
+ * Internal dependencies
+ */
+import { getLanguageSlugs } from '../../lib/i18n-utils/utils';
```

Comment:
This function is the same one used inside the abtest lib so we are sure we're covering all languages. See https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/abtest/index.js#L23